src/Applications/LDAS_App/util/average_ensemble_forcing.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+#
+# module load python/GEOSpyD/Ana2019.03_py3.7
+# module  load nco/4.8.1
+#
+import os
+import glob
+import subprocess as sp
+
+def averaging_forcing(in_path, out_path, nens):
+   """ The ensemble number will be appended to in_path
+       it starts from 001
+       out_path will be created if it does not exists """ 
+   if not os.path.exists(out_path):
+      os.makedirs(out_path) 
+   files_list=[]
+   for i in range(1,nens+1):
+      sfx = f'{i:03}'
```

Comment:
@saraqzhang :  I'm guessing that this line means we use 3 digits for the ensemble ID string.  Is this the practice in the ADAS?  In the LDAS, we've typically been using 4 digits, as in "ens0018", for example.  Unless there's some ADAS-related reason, it'd probably be better to use 4 digits for consistency with the rest of the LDAS.  I might be completely misinterpreting this piece of code, though.  Apologies if that's the case.

Reply:
@gmao-rreichle:  @weiyuan-jiang  ADAS atmens use 3 digits for member ID. this script is to access all member subdir in ADAS atmens to calculate ens average of the forcing.  Should we put some comments in the script specifying the script is for processing hybrid ADAS outputs.   If we wanted to make the script more general, we need to make user input argv for the digits, and the file name etc.

Reply:
@saraqzhang : Thanks for confirming that 3 digits is ADAS practice.  In this case it's fine to leave things as they are.  I suppose at a later time we should make these new py scripts more flexible.  As you put together the LADAS, you'll find out which variables would be good to have as inputs to the script, or at least have them set at the top of the script (rather than being buried in the code below).  For now let's move ahead with the PR.
cc: @weiyuan-jiang @biljanaorescanin